core/Makefile

```diff
@@ -114,10 +114,14 @@ CXXFLAGS += -std=c++17 -g3 -ggdb3 $(CXXARCHFLAGS) \
             -isystem $(dir $<).. -isystem $(COREDIR)/modules \
             -D_GNU_SOURCE \
             -Werror -Wall -Wextra -Wcast-align -Wno-error=deprecated-declarations \
+            -Wno-error=array-bounds \
```

Comment:
This suppression tends to make me a bit nervous ... I'll trust that @sangjinhan checked it though

Reply:
I think this was due to: https://github.com/NetSys/bess/issues/915

That was autogenerated protobuf code which we knows works, so it was safe to ignore it.